skimage/measure/tests/test_structural_similarity.py

```diff
@@ -1,11 +1,21 @@
+import os
 import numpy as np
-from numpy.testing import assert_equal, assert_raises
+import scipy.io
+from numpy.testing import (assert_equal, assert_raises, assert_almost_equal,
+                           assert_array_almost_equal)
 
 from skimage.measure import structural_similarity as ssim
-
+from skimage.measure._structural_similarity import (gaussian_filter2,
+                                                    _discard_edges)
+import skimage.data
+from skimage.io import imread
+from skimage import data_dir
 
 np.random.seed(1234)
 
+cam = skimage.data.camera()
+cam_noisy = skimage.data.load('camera_noisy.png')
```

Comment:
I agree that would be a better solution.  I don't know what the seed was when I originally generated that image, so I would need to recompute the Matlab and IPOL reference values hardcoded in the tests.